ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -311,12 +339,6 @@ def process(self, device, results, unused):
             if not image.get('id', None):
                 continue
 
-            # If it's a snapshot, rather than a normal image, ignore it for
-            # the time being.
-            if 'server' in image:
-                log.debug("Ignoring image %s" % image.get('name', ''))
-                continue
-
```

Comment:
Looking into this, I think you appear to be missing missing a relationship between snapshot and instance.

I think you were attempting to address that by counting snapshots as images (and thus relating them to instances), but you don't relate the images to snapshot components, so that doesn't work.  And even if it did, it would be combining two things- the image (snapshot) that an instance was built from, and the instance that the snapshot was built from.

So I think you need to add a direct snapshot to instance relationship (which instance is this a snapshot of), assuming openstack tracks that.

The other issue, that of instances that are built from snapshots rather than traditional instances.. I would suggest we need to talk about that a bit more.  Can you revert this change for the time being and we revisit that topic?  I am concerned that adding every snapshot to the instance list is going to bloat the system considerably with what are mostly duplicate components.  

I think the right solution to that may be somewhat complex and we need to talk it over, but that can be addressed in a separate pull request.